app/src/hooks/useStakingOptions.ts

```diff
@@ -95,19 +103,26 @@ async function fetchData(provider: AnchorProvider): Promise<SoParams[]> {
         continue;
       }
 
+      let strikeQuote = calculateStrikeQuoteAtomsPerBaseToken({
+        baseMint,
+        quoteMint,
+        lotSize: lotSize.toNumber(),
+        strikeQuoteAtomsPerLot: strike.toNumber(),
+      });
+
+      if (!isUSDCQuotePair(baseMint, quoteMint)) {
```

Comment:
Preferred solution would be to modify isUpsidePool() if neither base or quote contains USDC. The issue is that non-USDC quote assets flag that the pool is a "downside" or put and to invert the strike when it shouldn't